Anchor.toml

```diff
@@ -0,0 +1,9 @@
+[features]
```

Comment:
Please use https://github.com/marinade-finance/solana-js-utils/blob/main/packages/solana-test-utils/runner.ts - my lightweight implementation of anchor test for JS. (Yeah need to fix some problems with reinitializing solana ledger - now it can be done only manually by removing the folder)